app/Http/Controllers/BlogController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Blog;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class BlogController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
```

Comment:
why have this comment without content ? if not comment please remove

Reply:
i fixed